packages/@aws-cdk/aws-lambda-nodejs/README.md

```diff
@@ -53,10 +53,41 @@ new lambda.NodejsFunction(this, 'my-handler', {
 ```
 
 ### Configuring Parcel
-The `NodejsFunction` construct exposes some [Parcel](https://parceljs.org/) options via properties: `minify`, `sourceMaps`,
-`buildDir` and `cacheDir`.
+The `NodejsFunction` construct exposes some [Parcel](https://parceljs.org/) options via properties: `minify`, `sourceMaps` and `cacheDir`.
 
 Parcel transpiles your code (every internal module) with [@babel/preset-env](https://babeljs.io/docs/en/babel-preset-env) and uses the
 runtime version of your Lambda function as target.
 
 Configuring Babel with Parcel is possible via a `.babelrc` or a `babel` config in `package.json`.
+
+### Working with modules
+
+#### Externals
+By default, all node modules are bundled except for `aws-sdk`. This can be configured by specifying
+the `externalModules` prop.
+
+```ts
+new lambda.NodejsFunction(this, 'my-handler', {
+  externalModules: [
+    'aws-sdk', // Use the 'aws-sdk' available in the Lambda runtime
+    'cool-module', // 'cool-module' is already available in a Layer
+  ],
+});
+```
+
+#### Install modules
+By default, all node modules referenced in your Lambda code will be bundled by Parcel.
+Use the `installModules` prop to specify a list of modules that should not be bundled
```

Comment:
Should we perhaps rename this to `nodeModules` to reflect that this is where these guys will land?